client/src/utils/__tests__/jsonUtils.test.ts

```diff
@@ -317,3 +317,112 @@ describe("getValueAtPath", () => {
     expect(getValueAtPath(obj, ["users", "1", "name"])).toBe("Jane");
   });
 });
+
+describe("JsonSchemaType elicitation field support", () => {
+  const sampleSchema: JsonSchemaType = {
+    type: "object",
+    title: "User Info",
+    description: "User information form",
+    properties: {
+      name: {
+        type: "string",
+        title: "Full Name",
+        description: "Your full name",
+        minLength: 2,
+        maxLength: 50,
+        pattern: "^[A-Za-z\\s]+$",
+      },
+      email: {
+        type: "string",
+        format: "email",
+        title: "Email Address",
+      },
+      age: {
+        type: "integer",
+        minimum: 18,
+        maximum: 120,
+        default: 25,
+      },
+      role: {
+        type: "string",
+        enum: ["admin", "user", "guest"],
+        enumNames: ["Administrator", "User", "Guest"],
```

Comment:
`enumNames` [is not part of the JSONSchema spec](https://github.com/rjsf-team/react-jsonschema-form/issues/532). The [recommended way to describe](https://github.com/json-schema-org/json-schema-spec/issues/57) enum labels is with `oneOf` or `anyOf`.